packages/nocodb/src/cache/NocoCache.ts

```
@@ -129,6 +129,71 @@ export default class NocoCache {
+  public static async getHash(
+    key: string,
+  ): Promise<Record<string, string | number>> {
+    if (this.cacheDisabled) return Promise.resolve({});
+    return this.client.getHash(`${this.prefix}:${key}`);
+  }
+
+  public static async getHashField(
+    key: string,
+    field: string,
+  ): Promise<string> {
+    if (this.cacheDisabled) return Promise.resolve(null);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)